### PR TITLE
[News] 2020-09-24

### PR DESCRIPTION
**Pull request by suetsuguk**
- https://github.com/sider/devon_rex/blob/2.26.1/CHANGELOG.md
- https://github.com/sider/runners/blob/0.35.0/CHANGELOG.md

### CODE REVIEW

File: docs/news/2020.md [hunk 1]
```diff
@@ -11,6 +11,51 @@ This document describes notable changes on Sider in the year 2020.
...
+## Update environments and tools on September 18, 2020
+
+<time class="news-date" datetime="2020-09-18">September 18, 2020</time>
+
+We have updated the following environments and tools. Please each tool's document for details.
+
+Highlight:
+
+- The environment variable `EXTRA_CERTIFICATE` is newly supported. When this variable is specified, the runner imports the certificate on the system. This feature allows you to connect GitHub Enterprise Server running with a self-signed certificate.
+- The [`parallel`](../tools/cplusplus/cppcheck.md#parallel) option of Cppcheck is newly supported. This option allows you to run multiple jobs in parallel by using the `-j` option of the `cppcheck` command.
```
Comment (suetsuguk) [inline docs/news/2020.md:23]: > __[remark-lint]__ Link to unknown heading in `../tools/cplusplus/cppcheck.md`: `parallel` ([view](https://sider.review/gh/repos/142577083/pulls/502#issue-4624718))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `missing-heading-in-file` | `warn` |
>
> You can [close this issue](https://sider.review/gh/repos/142577083/pulls/502#issue-4624718) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

The note for cppcheck.md#parallel is being written in a different PR, and the PR is not merged yet.

**Comment by suetsuguk** [comment]
As for the highlights of this release, I didn't create notes for following changes because I didn't know how much impact they have. If we need to list some of them, please let me know.

- RuboCop Add standard to optional gem list #1479
- RuboCop Improve warning messages #1480
- Make generated Gemfile content log valid as Ruby #1485

File: docs/news/2020.md [hunk 2]
```diff
@@ -11,6 +11,51 @@ This document describes notable changes on Sider in the year 2020.
 
 ---
 
+## Update environments and tools on September 18, 2020
+
+<time class="news-date" datetime="2020-09-18">September 18, 2020</time>
```
Comment (ybiquitous) [inline docs/news/2020.md:16]: [suggestion] We cannot release today, so please update the date to an actual one. 🙏

**Review by ybiquitous — APPROVED** [review]
LGTM 👏

**Comment by ybiquitous** [comment]
> As for the highlights of this release, I didn't create notes for following changes because I didn't know how much impact they have. If we need to list some of them, please let me know.

I think we don't need to include these changes as highlight.

File: docs/news/2020.md [hunk 3]
```diff
@@ -11,6 +11,51 @@ This document describes notable changes on Sider in the year 2020.
...
+
+We have updated the following environments and tools. Please each tool's document for details.
```
Comment (yykamei) [inline docs/news/2020.md:18]: Perhaps, `read` is missing.

```suggestion
We have updated the following environments and tools. Please read each tool's documentation for details.
```

**Review by yykamei — APPROVED** [review]
LGTM

Would you update the Go version for tools used for Go, with the except for golangci-lint?
These are still written wit `Go 1.14.3`.

In addition, don't forget to update the date of the news.

File: docs/news/2020.md (hunk 3 above)
Comment (suetsuguk) [reply docs/news/2020.md]: Fixed. commit 2f8ed26d650ca6ecfc03abdde73dafa4c6fb44bd

**Comment by suetsuguk** [comment]
Thank you for pointing out.
I have updated the Go version for tools used for Go.
commit 5113fbe2b3b04329929d8ff7c987c8712742d74b

**Review by yykamei — APPROVED** [review]
Thank you. LGTM

**Comment by suetsuguk** [comment]
@ybiquitous We are going to include the latest release of PHPMD (2.9.0), so that I have added an entry for it. commit c9f7389698a635352f82f5a94582b69301971475.

**Pull request merged**